talk/owt/sdk/base/customizedvideodecoderproxy.cc

```diff
@@ -25,25 +25,25 @@ bool CustomizedVideoDecoderProxy::Configure(const Settings& codec_settings) {
   if (external_decoder_) {
     if (codec_type_ == kVideoCodecH264 &&
         external_decoder_->InitDecodeContext(VideoCodec::kH264)) {
-      return WEBRTC_VIDEO_CODEC_OK;
+      return true;
```

Comment:
This does not look good. Can we restructure this with switch (codec_type_ ) {case xxx: {return external_coder_->InitDecodeContext(yyy); ....},
instead of having so many duplicated code here.

Reply:
Done. Replaced with a map.